src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/Header/Header.tsx

```diff
@@ -0,0 +1,68 @@
+import { useFormContext } from 'react-hook-form';
+import { Slider } from '@mui/material';
+
+import { Svg } from 'shared/components';
+import {
+  theme,
+  StyledClearedButton,
+  StyledFlexColumn,
+  StyledFlexTopCenter,
+  StyledLabelBoldLarge,
+  StyledLabelLarge,
+} from 'shared/styles';
+
+import { HeaderProps } from './Header.types';
+import { StyledImg, StyledSliderPanelHeader } from './Header.styles';
+
+const commonButtonStyles = {
+  p: theme.spacing(1),
+  mr: theme.spacing(0.2),
+};
+
+export const Header = ({
+  name,
+  label,
+  isExpanded,
+  isMultiple,
+  onArrowClick,
+  onTrashClick,
+}: HeaderProps) => {
+  const { getValues } = useFormContext();
+
+  const { min, max, minLabel, maxLabel, minImage, maxImage } = getValues(name);
+
+  if (isExpanded)
+    return (
+      <StyledSliderPanelHeader isExpanded>
+        <StyledClearedButton onClick={onArrowClick} sx={commonButtonStyles}>
+          <Svg id={isExpanded ? 'navigate-up' : 'navigate-down'} />
+        </StyledClearedButton>
+        <StyledLabelBoldLarge>{label}</StyledLabelBoldLarge>
+        {isMultiple && (
+          <StyledClearedButton sx={commonButtonStyles} onClick={onTrashClick}>
+            <Svg id="trash" width="20" height="20" />
+          </StyledClearedButton>
+        )}
+      </StyledSliderPanelHeader>
+    );
+
+  return (
+    <StyledSliderPanelHeader>
+      <StyledClearedButton onClick={onArrowClick} sx={commonButtonStyles}>
+        <Svg id={isExpanded ? 'navigate-up' : 'navigate-down'} />
+      </StyledClearedButton>
+      <StyledLabelBoldLarge>{label}</StyledLabelBoldLarge>
+      <StyledFlexTopCenter sx={{ width: '100%', gap: '2.4rem' }}>
+        <StyledFlexColumn sx={{ alignItems: 'center' }}>
+          {<StyledLabelLarge>{minLabel}</StyledLabelLarge>}
```

Comment:
as I see in design the label must be under the image

Reply:
as was discussed, the label must be below the image